docs/content/en/configuration.md

```diff
@@ -340,6 +340,25 @@ const parseTxt = file => file.split('\n').map(line => line.trim())
 }
 ```
 
+#### Manipulating existing parsers
+You can change the default parsers! You get them when you provide a function:
+
+**Example**
+
+```js{}[nuxt.config.js]
+const parseTxt = file => file.split('\n').map(line => line.trim())
+
+// in Config:
+
+{
+  extendParser: (defaultParser) => ({
+	  ".md": file => postCompiler(defaultParser[".md"](preCompiler(file))),
```

Comment:
It's not entirely clear to me where `postCompiler` and `preCompiler` are coming from here. I suppose these are the custom functions that the user is supposed to define theirselves?

Reply:
Yes they are defined by the user.

Reply:
Can we make this more clear by adding an empty function declaration above or something like that?

```typescript
function preCompiler(file: string) {}
function postCompiler(parsedFile: any) {}

{
  extendParser: (defaultParser) => ({
    ".md": file => postCompiler(defaultParser[".md"](preCompiler(file))),
  })
}
```

Reply:
Also maybe we should call them `preParser` since that term is used throughout this functionality, rather than `preCompiler`.

Reply:
Yes I'm not happy with the docs. I will work on that later if you want you can write something :D